src/web/app/pipes/response-giver-email.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { TEAM_PREFIX } from '../../types/feedback-response-details';
+
+/**
+ * get moderated person's email for a response
+ */
+@Pipe({
+  name: 'responseGiverEmail',
+})
+export class ResponseGiverEmailPipe implements PipeTransform {
+
+  /**
+   * get moderated person's email for a response
+   */
+  transform(response: any, firstStudentEmailInTeams: {[key: string]: string}): string {
```

Comment:
You may want to write what is the key and what is the value of `firstStudentEmailInTeams `